README.md

```diff
@@ -175,6 +175,18 @@ sudo apt-get purge cmake
 sudo snap install cmake --classic
 ```
 
+
+
```

Comment:
This doesn't explain why.  Nor does it give the error message that this is attempting to solve.  Should this be done by everyone?  Or only docker users?  Or only docker users who run into some error?  Why wouldn't everyone want to make this change?  What about Windows users?